src/styles/product-card.scss

```diff
@@ -0,0 +1,95 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__poster {
+    width: 160px;
+    height: 134px;
+    background-color: $secondary-color;
+  }
+
+  &__img {
+    height: 134px;
+    width: 160px;
+  }
+
+  &__product-item {
+    flex-grow: 1;
+    padding-top: 40px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $base-color;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
+    padding-top: 4px;
+  }
+
+  &__rating {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $base-color;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-top: 16px;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-top: 24px;
+    line-height: 18px;
+
+    &-tag {
+      font-weight: 400;
+      font-size: 12px;
+      color: $secondary-color;
+    }
+
+    &-value {
+      font-weight: 700;
+      font-size: 16px;
+      color: $base-color;
+    }

```

Comment:
It's not a modifiers or elements of `price`, it's a product-card block elements.
 More detail in video.
[video, timing 4:15](https://mate.academy/learn/css-advanced/bem#/video/911)
```suggestion
    &__price-tag {
      font-weight: 400;
      font-size: 12px;
      color: $secondary-color;
    }

    &__price-value {
      font-weight: 700;
      font-size: 16px;
      color: $base-color;
    }

```
![Screenshot 2022-07-21 at 19 59 48](https://user-images.githubusercontent.com/17098736/180271524-a7f67f2a-8beb-4155-a497-95617757c49e.png)